website/source/docs/commands/monitor.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+layout: "docs"
+page_title: "Commands: monitor"
+sidebar_current: "docs-commands-monitor"
+description: >
+  Stream the logs of a running nomad agent.
+---
+
+# Command: monitor
+
+The `nomad monitor` command streams log messages for a given agent.
+
+## Usage
+
+```plaintext
+nomad monitor [options]
+```
+
+The nomad monitor command can be used to stream the logs of a running Nomad agent. Monitor will follow logs and exit when interrupted or until the remote agent quits.
```

Comment:
i think the policy is to limit these lines to 80 characters, but i can't actually find the policy...

Reply:
:+1: will update